kevm-pyk/src/kevm_pyk/kproj/evm-semantics/asm.md

```diff
@@ -105,6 +105,7 @@ Operator `#revOps` can be used to reverse a program.
     rule #asmOpCode( GASLIMIT       ) =>  69
     rule #asmOpCode( CHAINID        ) =>  70
     rule #asmOpCode( SELFBALANCE    ) =>  71
+    rule #asmOpCode( BLOBHASH       ) =>  73
```

Comment:
Hmm, not related to this PR  but it seems like we're missing a rule for `BASEFEE` here.

```suggestion
    rule #asmOpCode( SELFBALANCE    ) =>  71
    rule #asmOpCode( BASEFEE        ) =>  72
    rule #asmOpCode( BLOBHASH       ) =>  73
```

Reply:
Indeed, it seems we're not implementing [EIP-3198](https://eips.ethereum.org/EIPS/eip-3198)

Reply:
Can you double check this please? I can see the opcode being handled [here](https://github.com/runtimeverification/evm-semantics/blob/master/kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md?plain=1#L1083) , so I think it was just ommited in `#asmOpCode` rules. If it's unimplemented, then yes, we can have a new PR, otherwise I think we can just add it here.

Reply:
Yeah, the feature seems to be the case. Thanks for pointing that out! I'll add it here! Good catch!

Reply:
Done!